### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rishabh710**
Addition of contributing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1936,6 +1940,34 @@ $ brew install aquasecurity/trivy/trivy
...
+
+If you think you have found a bug please follow the instructions below.
+
+- Please spend a small amount of time giving due diligence to the issue tracker. Your issue might be a duplicate.
+- Open a new issue if a duplicate doesn't already exist.
+- Remember users might be searching for your issue in the future, so please give it a meaningful title to help others.
+
+### Features
+
+We also use the GitHub issue tracker to track feature requests. If you have an idea to make kube-bench even more awesome follow the steps below.
```
Comment (simar7) [inline README.md:1955]:
```suggestion
We also use the GitHub issue tracker to track feature requests.
```

File: README.md [hunk 2]
```diff
@@ -1936,6 +1940,34 @@ $ brew install aquasecurity/trivy/trivy
...
+
+- Open a new issue.
+- Remember users might be searching for your issue in the future, so please give it a meaningful title to helps others.
```
Comment (simar7) [inline README.md:1958]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1936,6 +1940,34 @@ $ brew install aquasecurity/trivy/trivy
...
+- Clearly define the use case, using concrete examples.
+- If you would like to include a technical design for your feature please feel free to do so.
+
+### Pull Requests
+
+We welcome pull requests!
+
+- Your PR is more likely to be accepted if it focuses on just one change.
+- Please include a comment with the results before and after your change.
+- Your PR is more likely to be accepted if it includes tests. (We have not historically been very strict about tests, but we would like to improve this!).
```
Comment (simar7) [inline README.md:1968]:
```suggestion
- Your PR is more likely to be accepted if it includes tests. If you're having difficulty with writing tests, please feel free to reach out to one of us and we'll be happy to guide you on to the right path.
```

File: README.md [hunk 4]
```diff
@@ -1936,6 +1940,34 @@ $ brew install aquasecurity/trivy/trivy
...
+- You're welcome to submit a draft PR if you would like early feedback on an idea or an approach.
+- Happy coding!
```
Comment (simar7) [inline README.md:1970]:
```suggestion
```

**Review by simar7 — CHANGES_REQUESTED** [review]

**Comment by simar7** [comment]
Please sign the CLA.

**Comment by knqyf263** [comment]
We already have [CONTRIBUTING.md](https://github.com/aquasecurity/trivy/blob/master/CONTRIBUTING.md)

**Comment by simar7** [comment]
I didn't even notice that. This PR should be closed in that case.

**Pull request closed without merging**